modules/virt-about-hostpath-provisioner.adoc

```diff
@@ -6,22 +6,8 @@
 [id="virt-about-hostpath-provisioner_{context}"]
 = About the hostpath provisioner
 
-The hostpath provisioner is a local storage provisioner designed for
-{VirtProductName}. If you want to configure local storage for
-virtual machines, you must enable the hostpath provisioner first.
+When you install the {VirtProductName} Operator, the hostpath provisioner (HPP) Operator is automatically installed. The HPP is a local storage provisioner designed for {VirtProductName}. If you want to configure local storage for virtual machines, you must enable the HPP first.
 
-When you install the {VirtProductName} Operator, the hostpath provisioner Operator
-is automatically installed. To use it, you must:
+In {VirtProductName} 4.10, the HPP is implemented using a Kubernetes CSI driver. The CSI driver enables you to configure local storage for your virtual machines with much less performance overhead, such as a restart of all {product-title} nodes because the `MachineConfig` object is no longer used to write to the directory on the host.
 
-* Configure SELinux:
-** If you use {op-system-first} 8 workers, you must create a `MachineConfig`
-object on each node.
-** Otherwise, apply the SELinux label `container_file_t` to the persistent volume (PV) backing
-directory on each node.
-* Create a `HostPathProvisioner` custom resource.
-* Create a `StorageClass` object for the hostpath provisioner.
-
-The hostpath provisioner Operator deploys the provisioner as a _DaemonSet_ on each
-node when you create its custom resource. In the custom resource file, you specify
-the backing directory for the persistent volumes that the hostpath provisioner
-creates.
+In {VirtProductName} 4.10, the HPP operator recognizes the legacy format of existing custom resources, and creates a legacy provisioner _DaemonSet_ as well as a CSI driver provisioner DaemonSet.
```

Comment:
```
In {VirtProductName} 4.10, the HPP operator recognizes the legacy format of existing custom resources, and creates a legacy provisioner as well as a CSI driver.
```
The daemonset part is not interesting to admins or users.

Reply:
Changed to:

In {VirtProductName} 4.10, the HPP Operator is implemented using a Kubernetes CSI driver, yet also recognizes the legacy format of existing custom resources.